lib/src/main/java/com/futurewei/alcor/common/repo/ICacheRepository.java

```diff
@@ -27,7 +27,7 @@
 
     Map<String, T> findAllItems() throws CacheException;
 
-    void addItem(T newItem) throws CacheException;
+    void addItem(T newItem) throws Exception;
```

Comment:
A general comment: If we could use a specific exception, it is actually preferred. So let us keep CacheException if possible.